mlflow/pyfunc/__init__.py

```diff
@@ -1047,10 +1054,21 @@ def spark_udf(spark, model_uri, result_type="double", env_manager="local"):
 
     if not should_use_spark_to_broadcast_file:
         # Prepare restored environment in driver side if possible.
+        # Note: In databricks runtime, because databricks notebook cell output cannot capture
+        # child process output, so that set capture_output to be True so that when conda prepare
+        # env command failed, the exception message will include command stdout/stderr output.
```

Comment:
```suggestion
        # child process output, so that set capture_output to be True so that when `conda prepare
        # env` command failed, the exception message will include command stdout/stderr output.
```